metadata/contributors.json

```diff
@@ -7445,4 +7453,4 @@
       ],
       "status" : "reviewer"
    }
-]
\ No newline at end of file
+]
```

Comment:
Please run `Tools/Scripts/validate-committer-lists --canonicalize` to fix up the formatting (and sort alphabetically by name).

Reply:
Just fixed! Thank you for pointing out!